packages-next/auth/src/types.ts

```diff
@@ -92,3 +93,12 @@ export type AuthTokenRedemptionErrorCode =
   | 'TOKEN_MISMATCH'
   | 'TOKEN_EXPIRED'
   | 'TOKEN_REDEEMED';
+
+type T = KeystoneListsAPI<Record<string, BaseGeneratedListTypes>>[string];
+type KeystoneList = { adminUILabels: { singular: string; plural: string } };
+export type Context = {
```

Comment:
You'll be able to use the type I've created in #4414 soon, has been bugging me that we re-type this everywhere